_security/access-control/document-level-security.md

```diff
@@ -36,6 +36,568 @@ This query specifies that for the role to have access to a document, its `genres
 A typical request to the `_search` API includes `{ "query": { ... } }` around the query, but in this case, you only need to specify the query itself.
 
 In the REST API, you provide the query as a string, so you must escape your quotes. This role allows a user to read any document in any index with the field `public` set to `true`:
+---
+layout: default
+title: Document-level security
+parent: Access control
+nav_order: 85
+redirect_from:
+- /security/access-control/document-level-security/
+- /security-plugin/access-control/document-level-security/
+---
+
+# Document-level security (DLS)
+
+Document-level security lets you restrict a role to a subset of documents in an index. The easiest way to get started with document- and field-level security is to open OpenSearch Dashboards and choose **Security**. Then choose **Roles**, create a new role, and review the **Index permissions** section.
+
+![Document- and field-level security screen in OpenSearch Dashboards]({{site.url}}{{site.baseurl}}/images/security-dls.png)
```

Comment:
@kolchfa-aws can you confirm if the hyphen syntax is correct here?

Reply:
It's a good observation, I was following the page as it was previously but it would be good to get another opinion on what is preferred please @kolchfa-aws